src/styles/LoginStyles.ts

```diff
@@ -32,12 +34,15 @@ export const LoginButton = styled(Button)`
     box-shadow: none;
     width: 187px;
     height: 48px;
-    margin: 8px 0px;
     text-transform: none;
+    display: flex;
```

Comment:
It doesn't look like you're using flex properties here -- if you're trying to get items to display on separate lines, you can do `display: block;`